package.json

```diff
@@ -34,13 +35,21 @@
     "@devtools-ds/object-inspector": "^1.1.2",
     "@devtools-ds/object-parser": "^1.2.1",
     "@devtools-ds/table": "^1.1.2",
-    "@devtools-ds/themes": "^1.1.2",
+    "@devtools-ds/themes": "^1.2.1",
```

Comment:
should this match the version for other `@devtools-ds` scoped packages?

Reply:
Yes. Updated.